### ISSUE TITLE
About CUB-200-2011's accuracy

### ISSUE THREAD

**Issue by shiyan-cui** [PKU-ICST-MIPL/SIM-Trans_ACMMM2022#1]
Thanks for your work and sharing your codes! I reproduce your code three times on RTX3090*4 entirely following the instruction,but I just got 91.4% as the best accuracy of CUB-200-2011.Could you analyze the problem about this? Thank you for your help。

**Comment by learn-in-practice** [comment]
Thanks for your interest to our work. As stated in the paper and the code introduction, we conduct all the experiments on NVIDIA GeForce GTX 1080 Ti GPUs with python 3.7.7, torch 1.5.0, torchvision 0.6.0, and apex package (installed without --cpp_ext). Different GPU platforms, python package versions, parallel operations, etc., may cause the fluctuation due to function calculation ways, parallel calculation order, and floating point precision. We rearrange the code to make it clear. Check the above content, retry the code, and perhaps slightly tune some training settings suitable to the code operation environment may help.
